internal/provider/resource_k8s.go

```diff
@@ -452,9 +491,9 @@ func (k *K8sDeployer) storeState(d *schema.ResourceData, cl *apiClient) {
 	for _, w := range k.Workers {
 		workers = append(workers, w.Dictify())
 	}
-	nodeDeploymentID := make(map[string]interface{})
-	for node, id := range k.NodeDeploymentID {
-		nodeDeploymentID[fmt.Sprintf("%d", node)] = int(id)
+	capacityDeploymentMap := make(map[string]interface{})
+	for capacityId, deploymentId := range k.CapacityDeploymentMap {
+		capacityDeploymentMap[fmt.Sprint("%d", capacityId)] = int(deploymentId)
```

Comment:
why do you need the formatting specifier for the Sprint? also please keep using 64 instead of ints

Reply:
terraform schema doesn't have uint64, so at the end all will  be treated as ints